src/main/java/com/mineinabyss/bonfire/listeners/PlayerListener.kt

```diff
@@ -43,11 +40,14 @@ import org.jetbrains.exposed.sql.deleteWhere
 import org.jetbrains.exposed.sql.innerJoin
 import org.jetbrains.exposed.sql.select
 import org.jetbrains.exposed.sql.transactions.transaction
+import java.util.*
 import kotlin.math.abs
 import kotlin.time.Duration.Companion.seconds
 
 object PlayerListener : Listener {
 
+    val cooldownMap = mutableListOf<UUID>()
```

Comment:
I think we already depend on geary here, could just add a component to the player right?